### PR TITLE
Implements the launchpad for the SNS V2

### PR DESCRIPTION
**Pull request by uk4z**

### CODE REVIEW

File: src/routes/launchpad/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+	$: status;
+</script>
+
+<main>
+	<div class="header-container">
+		<p>
+			WaterNeuron is a liquid staking protocol designed for the Internet Computer network. Staking
+			ICP becomes straightforward and efficient.
+		</p>
+		<span>https://docs.waterneuron.fi | https://x.com/waterneuron | https://t.me/waterneuron</span>
```
Comment (leo-ship-it) [inline src/routes/launchpad/+page.svelte:233]: If you want to put links, please use a tag.

File: src/routes/launchpad/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+				<span>Minimum Participant Commitment</span> <span>10 ICP</span>
+			</div>
+
+			<div class="parameter">
+				<span>Maximum Participant Commitment</span> <span>Unlimited</span>
+			</div>
+
+			<div class="parameter">
+				<span>Swap End</span> <span>Jul 3, 2024 6:00 PM</span>
+			</div>
```
Comment (leo-ship-it) [inline src/routes/launchpad/+page.svelte:238]: You can add swap start, and dynamically show the real date not a mock one.

File: src/routes/launchpad/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+			<div class="parameter">
+				<span>Token Symbol</span> <span>WTN</span>
+			</div>
+
+			<div class="parameter">
+				<span>Total Supply</span> <span>116'479'684.72 WTN</span>
+			</div>
+			<div class="parameter">
+				<span>Minimum Participants</span> <span>Unlimited</span>
+			</div>
```
Comment (leo-ship-it) [inline src/routes/launchpad/+page.svelte:226]: Is this relevant if there's no minimum.

File: src/routes/Toast.svelte [hunk 4]
```diff
@@ -14,7 +14,7 @@
 			<div class="info-container">
 				<div class="info-icon">
 					{#if toast.type == ToastType.Success}
-						<SuccessIcon />
+						<SuccessIcon color="--main-button-text-color" />
```
Comment (leo-ship-it) [inline src/routes/Toast.svelte:17]: `Type 'string' is not assignable to type 'never'.ts(2322)`

File: src/routes/launchpad/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,676 @@
...
+		const previousSnsCheckRatio = 65.6;
+		snsRatio = ratio;
+		selector.style.left = `${(ratio * previousSnsCheckpoint) / previousSnsCheckRatio}px`;
+		currentBar.style.width = `${(ratio * previousSnsCheckpoint) / previousSnsCheckRatio}px`;
+		minCommitment.style.left = `${barWidth / 2}px`;
+	};
+
+	const setUpSns = async () => {
+		const agent = HttpAgent.createSync({
+			host: 'http://127.0.1:8080'
```
Comment (leo-ship-it) [inline src/routes/launchpad/+page.svelte:97]: this is wrong

File: src/routes/launchpad/+page.svelte (hunk 3 above)
Comment (uk4z) [reply src/routes/launchpad/+page.svelte:226]: agreed

File: src/routes/launchpad/+page.svelte (hunk 2 above)
Comment (uk4z) [reply src/routes/launchpad/+page.svelte:238]: yes, done

File: src/routes/launchpad/+page.svelte (hunk 1 above)
Comment (uk4z) [reply src/routes/launchpad/+page.svelte:209]: done

**Review by uk4z — COMMENTED** [review]
wip review

**Review by uk4z — COMMENTED** [review]
ok

**Pull request merged**